CHANGELOG.md

```diff
@@ -302,7 +302,7 @@ Classify the change according to the following categories:
 - `proforma`: Updated storage to read per kW and per kWh incentives
 - `proforma`: Updated final cashflow to include non-taxed year 0 incentives (CBI and IBI)
 - `*.jl`: `MinChargeAdd` in `reopt.jl` was only accounting for year zero charges (needs to be lifecycle charges)
-- `REopt_Lite_API`: Use Django version 2.2.13 (had been 2.2.6)
+- `REopt_Lite`: Use Django version 2.2.13 (had been 2.2.6)
```

Comment:
@NLaws did all these replacements of `REopt_Lite_API` should be `REopt_API`, right? It looks like they were all replaced with `REopt_Lite`.

Reply:
right, should be `REopt_API`

Reply:
please do!

Reply:
Addressed in https://github.com/NREL/REopt_API/pull/296/commits/2f87b150df6e7c6c9188322ee40576decd17fdd7